rhosp10/profiles/cui.profile

```diff
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+title: 'RHOSP CUI'
```

Comment:
@redhat-rmcallis please change this to `[DRAFT] Controlled Unclassified Infomration (CUI) Profile for Red Hat OpenStack Plaform 10`

Reply:
changed